### DISCUSSION TITLE
Report Faulty or Missing Speeches

### DISCUSSION THREAD

**Discussion by JabobKrauskopf** [open-discourse/open-discourse#61] [General]
Please use this Thread to report missing or faulty Speeches.

Your Report should include:
- Release Version (or any unique identifier like a git commit hash)
- Speech ID
- Small Description

**Comment by FloLimebit** [comment]
- Release Version: `1.0.0`
- SpeechID: `1032169`
- Descr: Some parts of the spoken content are cut off. The bug occurs every time the word `CO2` is mentioned.
